src/click/termui.py

```diff
@@ -771,16 +773,11 @@ def pause(info: str | None = None, err: bool = False) -> None:
     if not isatty(sys.stdin) or not isatty(sys.stdout):
         return
 
-    if info is None:
-        info = _("Press any key to continue...")
```

Comment:
This is here for translation, which must happen dynamically, not in the default.

Reply:
NOW IT MAKES SENSE!

This makes me feel terrible about it😅